cmd/delete-oldsnapshots/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:1.19
```

Comment:
As you are running the binary it might be worth seeing if you can run this in a scratch container. This image build currently comes to ~1.54GB, it would be significantly smaller and also more secure if that were run from a container built from "scratch". Happy to pair with you on this :)